src/helpers.js

```diff
@@ -100,3 +101,13 @@ module.exports.logError = (error) => {
   console.info(chalk.rgb(...colors.red)(error));
   console.info('\n');
 }
+
+module.exports.logUndoConfirmation = (name) => {
+  console.info(`This will delete the component '${chalk.bold(name)}' and all files within the directory.`);
+  console.info(chalk.rgb(...colors.blue)(`Press ${chalk.bold`Enter`} to continue, or ${chalk.bold`any other key to cancel`}.`));
```

Comment:
What do you think about using the convention of `y/N`? Not sure if you've seen that, but you have to enter either 'y' for yes, or 'n' for no. It's case-insensitive, and the capital N indicates that if you just press 'enter', that will be selected. I think maybe any other key will also opt for the default N option.

You'll see below I suggest not prompting the user for components made in the last minute - because this will now be prompting for slightly older components, I think it's wise to make it a bit harder to delete a component :)